app/models/json_builders/channels_json_builder.rb

```diff
@@ -26,15 +26,15 @@ def initialize
   def build
     channels = []
 
-  [Channel.left_joins(publisher: :site_banner).verified.site_channels, Channel.left_joins(publisher: :site_banner).youtube_channels, Channel.left_joins(publisher: :site_banner).twitch_channels, Channel.left_joins(publisher: :site_banner).twitter_channels].each do |verified_channels|
+  [Channel.left_joins(publisher: :site_banners).verified.site_channels, Channel.left_joins(publisher: :site_banners).youtube_channels, Channel.left_joins(publisher: :site_banners).twitch_channels, Channel.left_joins(publisher: :site_banners).twitter_channels].each do |verified_channels|
       verified_channels.find_each do |verified_channel|
         if @excluded_channel_ids.include?(verified_channel.details.channel_identifier)
           excluded = true
           @excluded_verified_channel_ids.push(verified_channel.details.channel_identifier)
         else
           excluded = false
         end
-        channels.push([verified_channel.details.channel_identifier, true, excluded, verified_channel.publisher&.site_banner&.read_only_react_property || {}])
+        channels.push([verified_channel.details.channel_identifier, true, excluded, verified_channel.publisher&.site_banners || {}])
```

Comment:
It should only load the site banner for that channel. We can walk through this together.

Reply:
`verified_channel.publisher&.site_banners` is an ActiveRecord relation. The method `read_only_react_property` is meant to return the value stored for that site banner. Given that we'll have defaults, this should return the default version if there's not already a site banner for that channel.